### PR TITLE
nco v4.9.1

### PR DESCRIPTION
**Pull request by xylar**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
Not sure why the bot didn't notice the release yet...

File: .ci_support/win_c_compilervs2015cxx_compilervs2015vc14.yaml [hunk 1]
```diff
@@ -27,8 +27,6 @@ pin_run_as_build:
     max_pin: x
   expat:
     max_pin: x.x
-  libnetcdf:
-    max_pin: x.x.x
```
Comment (xylar) [inline .ci_support/win_c_compilervs2015cxx_compilervs2015vc14.yaml]: @ocefpaf, was this expected?  Are we no longer pinning `libnetcdf` run as build? Or is the bot glitching?  Either one isn't good news.

Comment (xylar) [reply .ci_support/win_c_compilervs2015cxx_compilervs2015vc14.yaml]: As I commented elsewhere, maybe it's fine.  https://abi-laboratory.pro/index.php?view=timeline&l=netcdf shows 100% compatibility going back many releases.

Comment (ocefpaf) [reply .ci_support/win_c_compilervs2015cxx_compilervs2015vc14.yaml]: In theory [this should](https://github.com/conda-forge/libnetcdf-feedstock/blob/099ecbb1f53e182ca874a5e50f05965b3123e47a/recipe/meta.yaml#L55) take care of that.

Comment (xylar) [reply .ci_support/win_c_compilervs2015cxx_compilervs2015vc14.yaml]: Ah, thanks for the explanation.

**Pull request merged**